plugin/aws/ec2InstanceConsoleOutput.go

```diff
@@ -28,25 +32,82 @@ func newEC2InstanceConsoleOutput(inst *ec2Instance, latest bool) *ec2InstanceCon
 	}
 	cl.DisableDefaultCaching()
 
-	return cl
+	output, err := cl.cachedConsoleOutput(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	attr := plugin.EntryAttributes{}
+	attr.SetCtime(output.mtime)
+	attr.SetMtime(output.mtime)
+	attr.SetAtime(output.mtime)
+	attr.SetSize(uint64(len(output.content)))
+	attr.SetMeta(output.toMeta())
+	cl.SetInitialAttributes(attr)
+
+	cl.Sync(plugin.CtimeAttr(), "LastModified")
```

Comment:
I was also toying with something like `Sync(key, ...SyncableAttrField)` but these seem like special enough cases that it isn't worth it (especially since it makes specifying a munge function difficult).

Reply:
Yeah, doesn't seem worth it.